core/src/main/scala/com/broadinstitute/dsp/models.scala

```diff
@@ -49,10 +53,46 @@ final case class KubernetesCluster(clusterName: KubernetesClusterName,
                                    location: Location) {
   override def toString: String = s"${googleProject}/${clusterName}"
 }
+
 final case class Nodepool(nodepoolId: Long,
                           nodepoolName: NodepoolName,
                           clusterName: KubernetesClusterName,
                           googleProject: GoogleProject,
                           location: Location) {
-  override def toString: String = s"${googleProject}/${nodepoolName}"
+  override def toString: String = s"$googleProject/${nodepoolName.value}"
+}
+
+final case class DeleteNodepoolMeesage(nodepoolId: Long, googleProject: GoogleProject, traceId: Option[TraceId]) {
+  val messageType: String = "deleteNodepool"
+}
+
+sealed abstract class RemovableStatus extends Product with Serializable {
```

Comment:
Do these `RemovableStatus`es only apply to nodepools? If so, should the name reflect that for clarity?